### DISCUSSION TITLE
How to increase the speed of processing?

### DISCUSSION THREAD

**Discussion by VMM-MMV** [mindee/doctr#1823] [Q&A] [answered]
I trying to compile the models, as per [this ](https://mindee.github.io/doctr/latest/using_doctr/using_model_export.html).

But I see no real difference in processing speed.

Are there some other ways to improve the speed of the models?

I don't particularly care about high accuracy, I just need the processing of **pdf to txt** to be fast.

**Comment by felixdittrich92** [comment]
Hi @VMM-MMV :wave:,

Then you should try: https://github.com/felixdittrich92/OnnxTR  :)

**Reply by felixdittrich92** [reply, marked as answer]
latency benchmark:

https://github.com/felixdittrich92/OnnxTR#benchmarks

I would suggest to try the standard installation `onnxtr[cpu]` first instead of `onnxtr[openvino]` because what i saw openvino differs heavily from CPU to CPU


with the following:
```
from onnxtr.models import ocr_predictor

model = ocr_predictor(
    det_arch='db_mobilenet_v3_large',  # detection architecture
    reco_arch='crnn_mobilenet_v3_small',  # recognition architecture
    det_bs=2, # detection batch size adjust to your hardware
    reco_bs=512, # recognition batch size adjust to your hardware
    )
```

You will get the fastest combination (maybe not the highest acc) ^^

**Comment by VMM-MMV** [comment]
Thank You. But, I am have some problems with it. Should I make a new discussion or should I ask here?

**Reply by felixdittrich92** [reply]
Opening a new would be good :+1:
